getting-started/templates/systemlink-values.yaml

```diff
@@ -84,6 +84,32 @@ global:
     ## CDN endpoint for telemetry-driven content. NI does not recommend changing this value.
     ##
     cdnEndpoint: "https://web-sdk.ni.com/"
+  ##
+  ## MongoDB Database Configuration.
+  ## Systemlink Enterprise' various services utilize MongoDB databases for data storage. Customers can deploy their own MongoDB instance
+  ## or utilize a managed solution such as MongoDB Atlas. Once provisioned provide the connection details in the values below.
+  ##
+  mongodb:
+    ## Enable or disable installation of per-service MongoDB instances. 
+    #  <ATTENTION> - If you do *not* have your own MongoDB instance set this (`global.mongodb.install`) to `true`.
+    ##
+    install: false 
+    ## <ATTENTION> - If `global.mongodb.install` is false, configure the external MongoDB connection string to be used by services in this deployment.
+    #
+    # If you are setting credentials per service (e.g. assetservice.secrets.mongodb.servicePassword) leave the placeholders
+    # <username> and <password> in the connection string. These will be replaced dynamically when deploying.
+    #
+    # If you have provisioned MongoDB to use a single set of credentials replace <username>:<password> with the values desired.
+    #
+    # <database> is required and will be replaced dynamically.
+    #
+    # examples:
+    # - MongoDB Atlas: mongodb+srv://<username>:<password>@my-atlas-cluster.mongodb.net/<database>
```

Comment:
I mentioned this on the AzDO PR too, but I think this should be using a secret instead of providing credentials directly in the values.yaml file.

Reply:
The `connection_string` has moved to `global.secrets.mongodb.connection_string`